lib/activity/processor.js

```diff
@@ -0,0 +1,6 @@
+module.exports = (logger) => {
```

Comment:
I think a comment here to this PR or a brief explanation of what this `processor` does would be awesome 💯

Reply:
A bit of documentation on your thought process would be helpful here and in the tests.